1-mandatory/1-writers.js

```diff
@@ -46,33 +46,53 @@ let writers = [
 
 /*
 Exercise 1:
-  Loop through the Array, and for each object, use `console.log()` to print out the below sentence
+  Loop through the Array, and for each writer, use `console.log()` to print out the below sentence
   and insert the corresponding values to the place holders that are indicated in curly braces:
-  "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
+  "Hi, my logDeadWriter is {firstlogDeadWriter} {lastlogDeadWriter}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
   // write your code to log all writers here
+
+
+  for (let writer of writers){
+
+  console.log(`Hi, my logDeadWriter is ${writer.firstlogDeadWriter} ${writer.firstlogDeadWriter}. I am ${writer.age} years old, and work as a ${writer.occupation}.`)
+   } // console.log(writers.map(
+  //   (writer) =>
+  //     `Hi, my logDeadWriter is ${writer.firstlogDeadWriter} ${writer.firstlogDeadWriter}. I am ${writer.age} years old, and work as a ${writer.occupation}.`
+  // ));
+  
 }
 
 /*
 Exercise 2:
   Only `console.log()` out the writers who are in their 40s (meaning between 40 and 49)
   and not alive anymore. Use the below sentence format:
-  "Writer {firstName} {lastName} died at {age} years old."
+  "Writer {firstlogDeadWriter} {lastlogDeadWriter} died at {age} years old."
 */
 
 function logDeadWritersInTheirForties() {
   // write your code here
-}
+  let logDeadWriters = writers.filter((writer) =>
```

Comment:
Filters are normally used to take items away - you are using the filter to do much more - I wonder if perhaps it would be easier to read the code if the filter only did filter and then you had a separate step to do the other actions?